src/ContainerLocator.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace League\Tactician\Container;
+
+use League\Container\Container;
+use League\Tactician\Container\Exception\MissingCommandException;
+use League\Tactician\Container\Exception\MissingContainerServiceException;
+use League\Tactician\Command;
+use League\Tactician\Handler\Locator\HandlerLocator;
+
+/**
+ * Fetch handler instances from an in-memory collection.
+ *
+ * This locator allows you to bind a handler fqcn to receive commands of a
+ * certain class name.
+ */
+class ContainerLocator implements HandlerLocator
+{
+    /**
+     * The Container object
+     *
+     * @var Container
+     */
+    protected $container;
+
+    /**
+     * The collection of Command/CommandHandler
+     *
+     * @var array
+     */
+    protected $commandToHandlerIdMap = [];
+
+    /**
+     * Class constructor
+     *
+     * @param Container $container                The Container object
+     * @param array     $commandClassToHandlerMap The Command/Handler mapping
+     */
+    public function __construct(
+        Container $container,
+        array $commandClassToHandlerMap = []
+    ) {
+        $this->container = $container;
+
+        if (empty($commandClassToHandlerMap) === false) {
```

Comment:
You can drop the empty check here. If it's not an array, the typehint will catch it. If it is an array, the addHandlers won't loop (and should always properly handle an empty array anyways)